recipes-extended/dpdk/dpdk.inc

```diff
@@ -27,92 +27,106 @@ export  MODULE_DIR = "/lib/modules/${KERNEL_VERSION}/kernel/drivers/net"
+    ### default value for prefix is "usr", unsetting it, so it
+    ### will not be concatenated in ${RTE_TARGET}/Makefile
+    ### which will cause compilation failure
+    #############################################################
+    unset prefix
+    oe_runmake O=$RTE_TARGET T=$RTE_TARGET config
 }
 
 do_compile () {
-	unset LDFLAGS TARGET_LDFLAGS BUILD_LDFLAGS
-
-	cd ${S}/${RTE_TARGET}
-	oe_runmake  CONFIG_RTE_EAL_IGB_UIO=n CONFIG_RTE_KNI_KMOD=y \
-	            CONFIG_RTE_LIBRTE_PMD_OPENSSL=y \
-                   EXTRA_LDFLAGS="-L${STAGING_LIBDIR} --hash-style=gnu" \
-		   EXTRA_CFLAGS="${HOST_CC_ARCH} ${TOOLCHAIN_OPTIONS} -I${STAGING_INCDIR}" \
-		   CROSS="${TARGET_PREFIX}" \
-		   prefix=""  LDFLAGS="${TUNE_LDARGS}"  WERROR_FLAGS="-w" V=1
-
-	cd ${S}/examples/
-        for APP in l2fwd l3fwd cmdif l2fwd-qdma l2fwd-crypto ipsec-secgw vhost kni ip_fragmentation ip_reassembly; do
-            temp=`basename ${APP}`
-            if [ ${temp} = "ipsec-secgw" ] || [ ${temp} = "l2fwd-crypto" ]; then
-	        oe_runmake EXTRA_LDFLAGS="-L${STAGING_LIBDIR} --hash-style=gnu -fuse-ld=bfd" \
-		       EXTRA_CFLAGS="${HOST_CC_ARCH} ${TOOLCHAIN_OPTIONS} -I${STAGING_INCDIR}" \
-		       CROSS="${TARGET_PREFIX}" -C ${APP} CONFIG_RTE_LIBRTE_PMD_OPENSSL=y O="${S}/examples/${temp}"
-            else
-                oe_runmake EXTRA_LDFLAGS="-L${STAGING_LIBDIR} --hash-style=gnu -fuse-ld=bfd" \
-                       EXTRA_CFLAGS="${HOST_CC_ARCH} ${TOOLCHAIN_OPTIONS} -I${STAGING_INCDIR}" \
-                       CROSS="${TARGET_PREFIX}" -C ${APP} CONFIG_RTE_LIBRTE_PMD_OPENSSL=y O="${S}/examples/${temp}/"
-            fi
-        done
-
+    unset LDFLAGS TARGET_LDFLAGS BUILD_LDFLAGS
+
+    cd ${S}/${RTE_TARGET}
+    oe_runmake \
+    CONFIG_RTE_EAL_IGB_UIO=n \
+    CONFIG_RTE_KNI_KMOD=y \
+    CONFIG_RTE_LIBRTE_PMD_OPENSSL=y \
+    EXTRA_LDFLAGS="-L${STAGING_LIBDIR} --hash-style=gnu" \
+    EXTRA_CFLAGS="${HOST_CC_ARCH} ${TOOLCHAIN_OPTIONS} -I${STAGING_INCDIR}" \
+    CROSS="${TARGET_PREFIX}" \
+    prefix="" LDFLAGS="${TUNE_LDARGS}" WERROR_FLAGS="-w" V=1
+
+    cd ${S}/examples/
+    for APP in l2fwd l3fwd cmdif l2fwd-qdma l2fwd-crypto ipsec-secgw vhost kni ip_fragmentation ip_reassembly; do
+        temp=`basename ${APP}`
+        if [ ${temp} = "ipsec-secgw" ] || [ ${temp} = "l2fwd-crypto" ]; then
+            oe_runmake \
+            EXTRA_LDFLAGS="-L${STAGING_LIBDIR} --hash-style=gnu -fuse-ld=bfd" \
+            EXTRA_CFLAGS="${HOST_CC_ARCH} ${TOOLCHAIN_OPTIONS} -I${STAGING_INCDIR}" \
+            CROSS="${TARGET_PREFIX}" -C ${APP} CONFIG_RTE_LIBRTE_PMD_OPENSSL=y O="${S}/examples/${temp}"
+        else
+            oe_runmake \
+            EXTRA_LDFLAGS="-L${STAGING_LIBDIR} --hash-style=gnu -fuse-ld=bfd" \
+            EXTRA_CFLAGS="${HOST_CC_ARCH} ${TOOLCHAIN_OPTIONS} -I${STAGING_INCDIR}" \
+            CROSS="${TARGET_PREFIX}" -C ${APP} CONFIG_RTE_LIBRTE_PMD_OPENSSL=y O="${S}/examples/${temp}/"
+        fi
+    done
```

Comment:
I re-read the style guide. It also say "Use spaces for indentation as developers tends to use different amount of spaces per one tab." I am OK for both:)

Reply:
I got commits rejected by OE-core guys using spaces for shell code. I can only encourage to avoid style changes to spaces for shell-code.
Using one style for all layers makes live easier ;)